AppleALC/ALCUserClientProvider/ALCUserClientProvider.cpp

```diff
@@ -0,0 +1,68 @@
+//
+//  ALCUserClientProvider.cpp
+//  AppleALC
+//
+//  Created by Nick on 10/14/20.
+//  Copyright © 2020 vit9696. All rights reserved.
+//
+
+#include "ALCUserClientProvider.hpp"
+
+OSDefineMetaClassAndStructors(ALCUserClientProvider, IOService);
+
+bool ALCUserClientProvider::start(IOService* provider) {
+	if (!super::start(provider))
+		return false;
+	
+	mHDACodecDevice = IOService::waitForMatchingService(IOService::nameMatching(kIOHDACodecDevice), 3000);	// Wait for 3s
+	
+	if (!mHDACodecDevice)
+	{
+		SYSLOG("client", "HDACodecDevice not found");
```

Comment:
Maybe it's better to make the timeout a debug log? And I wonder if it's necessary to call `stop`.

Reply:
Actually user should be aware that alc-verb will not work. So it is left as a SYSLOG. And no, it is not necessary to call stop as the init function returns prematurely in case of errors like above. Hence, that call is removed.